hudi-utilities/src/test/resources/schema/sample_data_schema.avsc

```diff
@@ -0,0 +1,10 @@
+{
+  "type": "record",
+  "name": "MySchema",
+  "fields": [
+    {
+      "name": "data",
+      "type": "string"
+    }
+  ]
+}
```

Comment:
nit: new line at the end
Also, license header at the top (though it's optional for .avsc files but as a convention we add here too).